### ISSUE TITLE
Questions about this repos

### ISSUE THREAD

**Issue by luna9722** [matterport/Mask_RCNN#2539]
Hello ,
I am new to Mask R-CNN and I am confused and don't know where to start to learn how to use this repos, what kind of annotations does it expect? does a dataset with only images and binary masks enough ?or should I have a json annotation (polygon segmentation with vgg annotator)? I didn't find a documentation or a good tutorial or a guideline that explains the basics. Please if someone have an idea where to start (some tutorials ...) and explains to me the type of expected annotations I'll be very grateful because I really need it for my academic project. Thanks in advance.

**Comment by rakesh-kumar-k** [comment]
Same here, if you get to know plz help!!

**Comment by EdoMW** [comment]
I strongly recommend following this tutorial:
Splash of Color: Instance Segmentation with Mask R-CNN and TensorFlow by Waleed Abdulla
and specifically go to the section about Training Dataset.

[https://engineering.matterport.com/splash-of-color-instance-segmentation-with-mask-r-cnn-and-tensorflow-7c761e238b46](url)
